Makefile

```
@@ -270,7 +270,8 @@ build-libsds: $(LIBSDS)
 
 build-libsds-android: clone-nim-sds
 	@echo "Building nim-sds for Android" $(LIBSDS)
-	$(MAKE) -C $(NIM_SDS_SOURCE_DIR) libsds-android ARCH=arm64 ANDROID_NDK_ROOT=$(ANDROID_NDK_ROOT) USE_SYSTEM_NIM=1 SHELL=/bin/bash
+	@if [ "$(ARCH)" != "arm64" ]; then SDSARCH=amd64; else SDSARCH=$(ARCH); fi; \
+	$(MAKE) -C $(NIM_SDS_SOURCE_DIR) libsds-android ARCH=$$SDSARCH ANDROID_NDK_ROOT=$(ANDROID_NDK_ROOT) USE_SYSTEM_NIM=1 SHELL=/bin/bash
```

Maybe? To keep consistency with others
```suggestion
	$(MAKE) -C $(NIM_SDS_SOURCE_DIR) libsds-android ARCH=$(SDSARCH) ANDROID_NDK_ROOT=$(ANDROID_NDK_ROOT) USE_SYSTEM_NIM=1 SHELL=/bin/bash
```